### ISSUE TITLE
Broader Monochrome UI Constraints

### ISSUE THREAD

**Issue by DexterStorey** [tedspare/hexle#9]
- With the monochrome UI PR: #1 there is no keyboard active ui state
- Some colors cannot achieve accessible contrast ratios with either black or white as the foreground contrast color
- More....?
